app/src/main/AndroidManifest.xml

```diff
@@ -41,6 +41,7 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.home.recentlyplayed.RecentlyPlayedActivity"
+      android:label="@string/recently_played_activity_label"
```

Comment:
Should we also be using this for the toolbar to ensure the two are kept in sync?